subsys/bluetooth/host/smp.c

```diff
@@ -374,6 +390,8 @@ static u8_t get_pair_method(struct bt_smp *smp, u8_t remote_io)
 	}
 
 	return gen_method_sc[remote_io][get_io_capa()];
+#endif
+	return JUST_WORKS;
```

Comment:
Shouldn't there be a `#else` branch here? Otherwise you'll generate code that has two consecutive unconditional return statements?

Reply:
Agree